src/lib/bristlefin/bristlefin.h

```diff
@@ -46,6 +46,10 @@ class Bristlefin {
   void enable3V();
   void disable3V();
 
+  // sdi
```

Comment:
@uma-kat - should make this more verbose, something like:
`// functions to switch between SDI12 Tx and Rx`